read-fonts/src/tables/glyf.rs

```diff
@@ -788,4 +828,68 @@ mod tests {
             ]
         );
     }
+
+    #[test]
+    fn compute_anchor_flags() {
+        let anchor = Anchor::Offset { x: -128, y: 127 };
+        assert_eq!(
+            anchor.compute_flags(),
+            CompositeGlyphFlags::ARGS_ARE_XY_VALUES
+        );
+
+        let anchor = Anchor::Offset { x: -129, y: 127 };
+        assert_eq!(
+            anchor.compute_flags(),
+            CompositeGlyphFlags::ARGS_ARE_XY_VALUES | CompositeGlyphFlags::ARG_1_AND_2_ARE_WORDS
+        );
+        let anchor = Anchor::Offset { x: -1, y: 128 };
+        assert_eq!(
+            anchor.compute_flags(),
+            CompositeGlyphFlags::ARGS_ARE_XY_VALUES | CompositeGlyphFlags::ARG_1_AND_2_ARE_WORDS
+        );
+
+        let anchor = Anchor::Point {
+            base: 255,
+            component: 20,
+        };
+        assert_eq!(anchor.compute_flags(), CompositeGlyphFlags::empty());
+
+        let anchor = Anchor::Point {
+            base: 256,
+            component: 20,
+        };
+        assert_eq!(
+            anchor.compute_flags(),
+            CompositeGlyphFlags::ARG_1_AND_2_ARE_WORDS
+        )
+    }
+
+    #[test]
+    fn compute_transform_flags() {
+        fn make_xform(xx: f32, yy: f32, xy: f32, yx: f32) -> Transform {
```

Comment:
the order of parameter to this `make_xform` helper looks odd to me, might as well follow the same order (xx, yx, xy, yy) in which these are defined in the Transform struct (which in turn follows the canonical column-major ordering of these)